src/app/[locale]/(components)/cartIconWithCount.tsx

```diff
@@ -2,14 +2,17 @@ import { getCheckoutID } from "@/core/server/checkout";
 import { gqlClient } from "@/gql";
 import { Suspense } from "react";
 import { CartIcon } from "./cartIcon";
+import { getTranslations } from "@/core/server/getTranslations";
 
 async function AsyncCartIcon({
   promise,
 }: {
   promise: Promise<number | undefined>;
 }) {
+  const t = await getTranslations();
```

Comment:
This should be done in parallel with the other promise instead of blocking

Reply:
done